### ISSUE TITLE
Uncaught No audio sink available. again?

### ISSUE THREAD

**Issue by robertjwhitney** [oampo/Audiolet#36]
hm. all of my previously working tests now seem to be returning this error... I see some changes to master head, and using this version removes the error, but my tests are silent. thoughts?

```
Uncaught No audio sink available. 
```

**Comment by oampo** [comment]
Which OS, browser and version is this with?  I'd hazard a guess that this is down to further changes filtering into the Web Audio API...

**Comment by robertjwhitney** [comment]
Oh word, sorry. OSX 10.7.3 Chrome 20.0.1132.57, and Also Safari 5.1.3

**Comment by oampo** [comment]
Sorry about the lack of progress Robert, I've been super busy and haven't had a moment to look at this.  Next week I should be able to get a free half-hour to see what's causing this.
Joe

**Comment by robertjwhitney** [comment]
Hey man! Thanks for checking in no problem, I've been super busy and feeling guilty about not getting to look into it also.

**Comment by oampo** [comment]
Okay - I've just tried this out with the latest trunk of Audiolet and the latest stable Chrome (21.0.1180.75) on Linux, and it is working okay here.  I can't find the package to downgrade to 20.0.1132.57, so would you mind upgrading to the latest stable and seeing whether that helps.  I'm using the examples/demo app to test it out, which seems to be working fine.  Otherwise I guess this is probably be a problem with the OSX version, and I'll need to get you to do a bit more debugging for me.

**Comment by robertjwhitney** [comment]
I think this is likely a conflict between sink.js and webkit at the moment.

https://github.com/jussi-kalliokoski/sink.js

Related 

https://github.com/jussi-kalliokoski/audiolib.js/issues/69
https://github.com/jussi-kalliokoski/audiolib.js/issues/60

I also noted that the blobBuilder syntax is deprecated in favor of blob, but that some things had changed with slice.

https://developer.mozilla.org/en-US/docs/DOM/Blob#Browser_compatibility

**Comment by robertjwhitney** [comment]
Hm. Actually seems fixed in latest chrome. Looking at Safari again.

**Comment by robertjwhitney** [comment]
Hmmm. Still not working in 5.1.6 Safari. Hopefully resolved soon.

**Comment by oampo** [comment]
Yeah, I'm pretty sure this will have been a change with the Web Audio API which has been slowly propagating through.  I don't have any way of testing in Safari at the moment, so if you could try this out in a version or so just to make sure there's nothing more serious that would be much appreciated.  Cheers for the work debugging this too :)

**Comment by bennlich** [comment]
Hmm. In Firefox 30 I'm getting `Error: No audio sink available. Audiolet.js:6368`, and in Chrome 36.0.1985.84 beta I'm getting `Uncaught [object Object] Audiolet.js:5991`. Sounds like maybe this is a known problem?

**Comment by Kamasoutra** [comment]
Fixed in my pull request (#58) :)

**Comment by bennlich** [comment]
Hot dawg! Awesome!
